src/components/Header.jsx

```diff
@@ -31,9 +46,17 @@ const Header = ({
   const { setRedirectToLogout, setRedirectToPage } = useRedirectHook()
   const { retrieveStagingUrl } = useSiteUrlHook()
   const [userId] = useLocalStorage(LOCAL_STORAGE_KEYS.GithubId)
+  const {
+    isOpen: isWarningOpen,
+    onOpen: onWarningOpen,
+    onClose: onWarningClose,
+  } = useDisclosure()
+  const {
+    isOpen: isStagingOpen,
+    onOpen: onStagingOpen,
+    onClose: onStagingClose,
+  } = useDisclosure()
```

Comment:
nit: could we just add the `modal` into the variable names? (eg: `isWarningModalOpen/isStagingModalOpen`)

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/941/commits/0721a811504d1b7b38d4d6eb61364e11db356000